core-blocks/columns/style.scss

```diff
@@ -1,9 +1,12 @@
 .wp-block-columns {
-	display: grid;
-	grid-auto-flow: dense;
+
+	.editor-inner-blocks {
+		display: grid;
+		grid-auto-flow: dense;
+	}
 
 	@for $i from 2 through 6 {
-		&.has-#{ $i }-columns {
+		&.has-#{ $i }-columns .editor-inner-blocks {
```

Comment:
This change seems to have inadvertently caused #7127, where the stylesheet is used for both the editor and front-end, but the markup generated from the front-end will not have `.editor-inner-blocks`.

Reply:
Looking.